src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Rename/RenameService.cs

```diff
@@ -288,6 +273,38 @@ private static async Task<ImmutableArray<TagHelperDescriptor>> GetOriginTagHelpe
         return [primaryTagHelper, associatedTagHelper];
     }
 
+    private static TagHelperBinding? TryGetTagHelperBinding(RazorSyntaxNode owner, int absoluteIndex)
```

Comment:
Consider: If it's not an async method, I'll usually try to change the signature of `TryGet` methods to return bool with an out parameter.